docs/rules/object-curly-newline.md

```diff
@@ -16,7 +16,8 @@ This rule has either a string option:
 Or an object option:
 
 * `"multiline": true` (default) requires line breaks if there are line breaks inside properties or between properties
-* `"minProperties"` requires line breaks if the number of properties is more than the given integer
+* `"minProperties"` requires line breaks if the number of properties is more than the given integer. By default, an error will also be reported if an object contains linebreaks and has fewer properties than the given integer. However, the second behavior is disabled if the `consistent` option is set to `true`
```

Comment:
@kaicataldo @valorkin  also i think we need clarify `more than` (in wiki it is https://en.wikipedia.org/wiki/Greater-than_sign), but in our code it is `node.properties.length >= options.minProperties` (`>=`)